tests/gateway/test_fluent.py

```diff
@@ -270,6 +270,26 @@ def test_get_route_accepts_unknown_provider():
         }
 
 
+def test_get_route_accepts_unknown_route_type():
+    set_gateway_uri("http://localhost:5000")
+    mock_resp = mock.Mock(status_code=200)
+    mock_resp.json.return_value = {
+        "name": "chat",
+        "route_type": "llm/v1/unknown",
+        "model": {"name": "gpt4", "provider": "openai"},
+        "route_url": "http://localhost:5000/gateway/chat/invocations",
+    }
```

Comment:
Can we assign this dictionary to a variable, as it's used down below as well 'assert route.dict() == ...'?

Reply:
@serena-ruan Sounds good, fixed!